bot/common/threads/utils.py

```diff
@@ -0,0 +1,68 @@
+import json
+import hashlib
+from bot.common.threads.thread_builder import ThreadKeys, build_cache_value
+from bot.common.threads.onboarding import Onboarding  # noqa: E402
+from bot.common.threads.update import UpdateProfile  # noqa: E402
+from bot.common.threads.initial_contribution import InitialContributions
+from bot.common.threads.report import Report
+from bot.common.threads.points import Points
+from bot.common.threads.add_dao import AddDao
+
+
+async def get_jump_thread(parent_thread, message, user_id):
```

Comment:
moved some common thread manip functions to this common file, rather than having them tied to specific threads.